unattended_scripts/unattended_installation.sh

```diff
@@ -51,10 +51,16 @@ importFunction() {
 }
 
 main() {
+
+    importFunction "common.sh"
+    importFunction "wazuh-cert-tool.sh"
+
     if [ "$EUID" -ne 0 ]; then
         echo "Error: This script must be run as root."
         exit 1;
-    fi   
+    fi
+
+    checkArch
```

Comment:
Call this after import the common.sh file

Reply:
This was fixed in the new commit.